bin.src/transd.py

```diff
@@ -81,7 +102,9 @@ def background_thread(cmd, pause=1):
         time.sleep(pause)
 
 
-if __name__ == "__main__":
+def main():
```

Comment:
In DM-22677, besides turning the daf_butler's drivers into functions, the functions were moved to the python directory and unit tests written.   I'm having problems finding the driver behind the changes (LSST developer guide, RFC, etc).